domains/zester.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "sxtxnzester",
+           "email": "lollo11020210@gmail.com",
+           "discord": "1133362142461050980"
+        },
+    
+        "record": {
+            "CNAME": "zester.github.io"
```

Comment:
Cannot access website

Reply:
the CNAME is wrong so u can delete it
the right one is sxtxnzester.github.io